raft/raft.go

```diff
@@ -654,6 +671,11 @@ func stepLeader(r *raft, m pb.Message) {
 					// an update before, send it now.
 					r.sendAppend(m.From)
 				}
+				// Transfer leadership is in progress.
+				if m.From == r.leadTransferee && pr.Match == r.raftLog.lastIndex() && !r.timeoutNowSent {
+					r.logger.Infof("%x sent MsgTimeoutNow to %x after received MsgAppResp", r.id, m.From)
```

Comment:
I feel this is an optimization. We can think this more and do it in another pr. @es-chow Can you set up another issue to track this potential optimization?